ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-presubmits.yaml

```diff
@@ -221,3 +221,71 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )unit,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/upgrade-full-integ-aws
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-origin-aggregated-logging-master-upgrade-full-integ-aws
+    rerun_command: /test upgrade-full-integ-aws
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/upgrade-full-integ-aws-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=upgrade-full-integ-aws
+        - --template=/usr/local/upgrade-full-integ-aws
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-origin-aggregated-logging-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: upgrade-full-integ-aws
+        - name: TEST_COMMAND
+          value: |
+            OPENSHIFT_BUILD_NAMESPACE= IMAGE_FORMAT= USE_CUSTOM_IMAGES=true hack/deploy-logging.sh
+            export LOGGING_DEPLOY_MODE=upgrade
+            . openshift/ci-operator/build-image/setup-logging-for-e2e.sh
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/upgrade-full-integ-aws-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/upgrade-full-integ-aws
+          name: job-definition
+          subPath: cluster-launch-installer-src.yaml
```

Comment:
just out of curiosity, do we have any other jobs that use this job template?

Reply:
yes - the `full-integ-aws` test uses this template